src/stories/2-frontend-engine/conditional-rendering/conditional-rendering-array.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import { Description, Stories, Title } from "@storybook/addon-docs";
```

Comment:
note: in terms of stories, it is mostly:
- created conditional rendering stories with some separated by types (arrays, numbers, strings)
- split existing validation schema stories by types (arrays, conditional, numbers, strings)